ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -917,7 +917,7 @@ def create_test_case_parameter_values(self, dbt_test):
         values = manifest_node.test_metadata.kwargs.get("values")
```

Comment:
maybe adding a typing List[str] be a better idea? wdyt? it will help clarify things in the below code

Reply:
This `values` list can be of any type not necessarily str. So adding List[str] would not be the right thing here